pkg/clients/ec2/vpc.go

```diff
@@ -39,9 +47,35 @@ func IsVPCNotFoundErr(err error) bool {
 	return false
 }
 
-// IsUpToDate returns true if there is no update-able difference between desired
+// IsVpcUpToDate returns true if there is no update-able difference between desired
 // and observed state of the resource.
-func IsUpToDate(spec v1alpha3.VPCParameters, o ec2.Vpc) bool {
-	actual := v1alpha3.BuildFromEC2Tags(o.Tags)
-	return cmp.Equal(spec.Tags, actual)
+func IsVpcUpToDate(spec v1beta1.VPCParameters, o ec2.Vpc) bool {
+	v1beta1.SortTags(spec.Tags, o.Tags)
+	actual := v1beta1.BuildFromEC2Tags(o.Tags)
+	return cmp.Equal(spec.Tags, actual) && (aws.StringValue(spec.InstanceTenancy) == string(o.InstanceTenancy))
+}
+
+// GenerateVpcObservation is used to produce v1beta1.VPCObservation from
+// ec2.Vpc.
+func GenerateVpcObservation(vpc ec2.Vpc) v1beta1.VPCObservation {
+	o := v1beta1.VPCObservation{
```

Comment:
As far as I can tell, the following fields exist in `ec2.Vpc` and can be included in `VPCObservation`:
```
// Information about the IPv4 CIDR blocks associated with the VPC.
CidrBlockAssociationSet []VpcCidrBlockAssociation `locationName:"cidrBlockAssociationSet" locationNameList:"item" type:"list"`

// The ID of the set of DHCP options you've associated with the VPC (or default
// if the default options are associated with the VPC).
DhcpOptionsId *string `locationName:"dhcpOptionsId" type:"string"`

// Information about the IPv6 CIDR blocks associated with the VPC.
Ipv6CidrBlockAssociationSet []VpcIpv6CidrBlockAssociation `locationName:"ipv6CidrBlockAssociationSet" locationNameList:"item" type:"list"`
```

Reply:
Have added these fields